log4j-core/src/main/java/org/apache/logging/log4j/core/LoggerContext.java

```diff
@@ -515,25 +522,23 @@ public Collection<Logger> getLoggers() {
     }
 
     /**
-     * Obtains a Logger from the Context.
+     * Obtains a logger from the context.
      *
-     * @param name The name of the Logger to return.
-     * @param messageFactory The message factory is used only when creating a logger, subsequent use does not change the
-     *            logger but will log a warning if mismatched.
-     * @return The Logger.
+     * @param name a logger name
+     * @param messageFactory a message factory to associate the logger with
+     * @return a logger matching the given name and message factory
      */
     @Override
     public Logger getLogger(final String name, final MessageFactory messageFactory) {
```

Comment:
_Nit_ regarding `getLogger(String)`: that method could use:

```java
return getLogger(name, DEFAULT_MESSAGE_FACTORY);
```

instead of

```java
return getLogger(name, null);
```

If we enable nullability warnings, this change will remove one.

Reply:
Fixed in 8a89e8c3903d4a5836459f19c6500938fd89b219.